src/components/ListItem.jsx

```diff
@@ -18,17 +18,24 @@ export function ListItem({ item, listToken }) {
 		}
 	};
 
+	const isRecentlyPurchased =
+		sub(new Date(), { days: 1 }) < item.dateLastPurchased;
+
 	return (
-		<li className="ListItem">
+		<li
+			className="ListItem"
+			title={isRecentlyPurchased && 'You recently purchased this item'}
+		>
 			<label htmlFor={item.id}>
 				<input
 					type="checkbox"
 					id={item.id}
-					defaultChecked={sub(new Date(), { days: 1 }) < item.dateLastPurchased}
-					disabled={sub(new Date(), { days: 1 }) < item.dateLastPurchased}
+					defaultChecked={isRecentlyPurchased}
+					disabled={isRecentlyPurchased}
 					onChange={(e) => {
 						handleUpdate(e.target.checked);
 					}}
+					title={isRecentlyPurchased && 'You recently purchased this item'}
```

Comment:
<img width="868" alt="image" src="https://user-images.githubusercontent.com/17270662/235189195-d30efa6d-f0e6-401f-8163-3e03fdac2173.png">

Reply:
Ah missed that. It's resolved